src/components/Footer/index.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { Header } from "../";
+import {
+  Container,
+  Row,
+  Link,
+  Logo,
+  Center,
+  Col,
+  Subheading,
+  Heading,
+  TextLink,
+  Icon,
+  TextHeight,
+} from "./styles";
+
+import { ReactComponent as Meta } from "../../assets/media/meta.svg";
+import { ReactComponent as Discord } from "../../assets/media/discord.svg";
+import { ReactComponent as Tiktok } from "../../assets/media/tiktok.svg";
+import { ReactComponent as Instagram } from "../../assets/media/instagram.svg";
+import CKI_Logo from "../../assets/images/cki_logo.png";
+
+function HyperLink(props) {
+  const Icon = props.icon;
+  const link = props.href;
+  return (
+    <li>
+      <Link href={link} target="_blank">
+        <Icon />
+        <div>{props.children}</div>
+      </Link>
+    </li>
+  );
+}
+
+export function Footer() {
+  return (
+    <div className="main-footer">
+      <Container>
+        <Row>
+          {/*column1 */}
+          <ul className="list-unstyled">
+            <Logo src={CKI_Logo} alt="Circle K International Logo" />
+            <Heading>Circle K</Heading>
+            <Subheading>Cal Poly Pomona</Subheading>
+          </ul>
+        </Row>
+        {/* Column2 */}
+        <Row>
+          <Col>
+            <Subheading>
+              <li> Contact Us </li>
+            </Subheading>
+            <p>
+              <Instagram fill="#B1953A" />
+              <TextLink href="https://Instagram.com" target="_blank">
+                cppcki
+              </TextLink>
+            </p>
+            <p>
+              <Meta fill="#B1953A" />
+              <TextLink href="https://Facebook.com" target="_blank">
```

Comment:
The icons could be a bit smaller to match the responding text's font size but 2px bigger.

Reply:
based on this, i can either lower the size of the icon to by 2, 28 to 26, or increase the size of the text from 20 to 22 based on your recommendation, which would be better?

Reply:
Don't increase the font-size. Decrease the SVG size instead.

Reply:
Font-size was needed since I used <p></p> for the spacing aspect, I don't really know of any other alternatives.

Reply:
You can add padding/margins explicitly. You don't have to depend on the implicit styles.